charts/krane/Chart.yaml

```diff
@@ -2,10 +2,10 @@ apiVersion: v2
 name: krane
 description: A Helm chart for Krane
 type: application
-version: 0.1.1
-appVersion: "v0.1.1"
+version: 0.1.2
+appVersion: "v0.1.2"
```

Comment:
Please stick to the v0.1.1 for now. As soon as new release goes out I'll make sure to bump it up. Thanks.